Application/EdFi.Ods.Api/Providers/XsdFileInformationProvider.cs

```diff
@@ -24,6 +24,7 @@ public class XsdFileInformationProvider : IXsdFileInformationProvider
         private readonly ILog _logger = LogManager.GetLogger(typeof(XsdFileInformationProvider));
         private readonly ISchemaNameMapProvider _schemaNameMapProvider;
         private readonly Lazy<IDictionary<string, XsdFileInformation>> _xsdFileInformationByUriSegment;
+        private const string CoreUriSegment = "ed-fi";
```

Comment:
EdFiConventions.UriSegment is what you want, not a new constant.

Reply:
done